app/travel_processor/ui5.yaml

```diff
@@ -30,3 +34,11 @@ server:
         baseUri: http://localhost:4004/processor
         username: admin # dummy credentials for local testing
         password: admin # dummy credentials for local testing
+    - name: fiori-tools-appreload
```

Comment:
same. is `ui5-middleware-simpleproxy` needed anymore?

Reply:
Same here, the app won't start properly without having it in the ui5.yaml as a custom middleware.